source/javascripts/components/unified-editor/StepBundleDrawer/StepBundlePropertiesTab.tsx

```diff
@@ -1,31 +1,56 @@
+import { Button, useDisclosure } from '@bitrise/bitkit';
 import EditableInput from '@/components/EditableInput/EditableInput';
 import StepBundleService from '@/core/models/StepBundleService';
 import { useStepBundles } from '@/hooks/useStepBundles';
-import useRenameStepBundle from './hooks/useRenameStepBundle';
+import DeleteStepBundleDialog from '@/pages/StepBundlesPage/components/DeleteStepBundleDialog/DeleteStepBundleDialog';
```

Comment:
Don't reference components under from a page, move the component to here instead